index.d.ts

```diff
@@ -276,7 +276,7 @@ export class Gauge<T extends string> {
 	 * @param labels Object with label keys and values
 	 * @return Function to invoke when timer should be stopped
 	 */
-	startTimer(labels?: LabelValues<T>): (labels?: LabelValues<T>) => void;
+	startTimer(labels?: LabelValues<T>): (labels?: LabelValues<T>) => number;
```

Comment:
I'm not sure that this is correct. From testing, reading the code and comparing to the other definitions in index.d.ts, it looks like this is correct:

```ts
startTimer(labels?: LabelValues<T>): (labels?: LabelValues<T>) => void;
```

Reply:
Ah I see part of the confusion: The code you linked to in your top post is for Histogram -- this is the Gauge class.

Reply:
Ah whoops. 🤦‍♂️